src/ProjNet/IO/CoordinateSystems/CoordinateSystemWktReader.cs

```diff
@@ -303,34 +304,46 @@ private static IProjection ReadProjection(WktStreamTokenizer tokenizer)
             else
                 tokenizer.CheckCloser(bracket);
 
-            tokenizer.ReadToken(",");//,
-            tokenizer.ReadToken("PARAMETER");
             var paramList = new List<ProjectionParameter>();
-            while (tokenizer.GetStringValue() == "PARAMETER")
-            {
-                bracket = tokenizer.ReadOpener();
-                string paramName = tokenizer.ReadDoubleQuotedWord();
-                tokenizer.ReadToken(",");
-                tokenizer.NextToken();
-                double paramValue = tokenizer.GetNumericValue();
-                tokenizer.ReadCloser(bracket);
-                paramList.Add(new ProjectionParameter(paramName, paramValue));
-                //tokenizer.ReadToken(",");
-                //tokenizer.NextToken();
-                tokenizer.NextToken();
-                if (tokenizer.GetStringValue() == ",")
-                {
-                    tokenizer.NextToken();
-                }
-                else
-                {
-                    break;
-                }
-            }
+
             var projection = new Projection(projectionName, paramList, projectionName, authority, authorityCode, string.Empty, string.Empty, string.Empty);
             return projection;
         }
 
+        private static void ReadParamater(WktStreamTokenizer tokenizer, Projection projection)
```

Comment:
Typo `ReadParameter`, argument `projection` could be of type `ICollection<ProjectionParameter>`